main.go

```diff
@@ -74,6 +75,7 @@ func main() {
 	super := initGD2Supervisor()
 	super.ServeBackground()
 	super.Add(servers.New())
+	super.Add(mgmt.New())
```

Comment:
I'd prefer this be done only when some config option (call it --mgmt or something) is turned on and not by default.

Even better if build tags can be used to conditionally compile and use `mgmt`. If this can be done easily, I'm okay for mgmt to be compiled by default while maintaining an option to exclude compiling it.